src/app/workflow/launch/launch.component.html

```diff
@@ -71,16 +71,26 @@
             {{ cwlrunnerDescription }}
             <pre>{{ cwl }}</pre>
           </div>
-          <!-- cwltool instructions for workflows will need to wait for https://github.com/common-workflow-language/cwltool/pull/1093
+        </mat-card>
+        <mat-card>
+          <!--          cwltool instructions for workflows will need to wait for https://github.com/common-workflow-language/cwltool/pull/1093-->
           <div [matTooltip]="cwltoolTooltip">
-            Alternatively, <a href="https://github.com/common-workflow-language/cwltool">cwltool</a> can conveniently
-            run
-            a workflow off of dockstore.org and other <a href="https://github.com/common-workflow-language/cwltool#use-with-ga4gh-tool-registry-api">GA4GH
-              tool registries</a> when all inputs and outputs are available on the local filesystem or via HTTP(S):
-            <pre>{{ dockstoreSupportedCwlLaunch }}</pre> cwltool can also be used to create the input template:
+            Alternatively, <a href="https://github.com/common-workflow-language/cwltool">cwltool</a> can conveniently run a workflow off of
+            dockstore.org and other
+            <a href="https://github.com/common-workflow-language/cwltool#use-with-ga4gh-tool-registry-api">GA4GH tool registries</a> when
+            all inputs and outputs are available on the local filesystem or via HTTP(S):
+            <pre>{{ dockstoreSupportedCwlLaunch }}</pre>
+            cwltool can also be used to create the input template:
             <pre>{{ dockstoreSupportedCwlMakeTemplate }}</pre>
           </div>
-          -->
+        </mat-card>
+        <mat-card
+          class="m-2"
+          matTooltip="Commands for launching tool through Consonance: a DIY open-source framework for
+      spinning up VMs and running tools on AWS EC2"
+        >
+          <a href="https://github.com/Consonance/consonance/wiki">Run on AWS VMs with Consonance</a>
+          <pre>{{ consonance }}</pre>
```

Comment:
not sure what this has to do with github apps

Reply:
Not sure either, we should comment this out if it hasn't already though.